src/Modal.scss

```diff
@@ -23,9 +23,15 @@
 
   input {
     font-size: 1em;
+    border: 2px solid transparent;
     border-radius: 12px;
     padding: var(--spacing-1);
   }
+
+  &:focus-visible {
+    border: 3px solid $rpf-brand-raspberry;
```

Comment:
Is this in the designs somewhere? I guess I'm wondering whether this should match the outline around the project renaming input field or the focus visible for buttons/links etc